erpnext/www/support/index.html

```diff
@@ -0,0 +1,155 @@
+				<input name='q' class="form-control py-2 border-right-0 border" type="Text"
+				value='{{ frappe.form_dict.q or ''}}'
+				{% if not frappe.form_dict.q %}placeholder="{{ _("Search for help articles and more") }}"{% endif %}>
+				<span class="input-group-append">
+					<div class="border input-group-text bg-white"><i class="fa fa-search text-muted"></i></div>
+				</span>
+				</form>
+			</div>
+		</p>
+	</div>
+</section>
+
+{% if favorite_article_list %}
+<div class='container mb-5'>
+	<h3 class="h3 pb-3">{{ _("Frequently Read Articles") }}</h3>
+	<div class="row row-cols-1 row-cols-md-3">
+		{% for favorite_article in favorite_article_list %}
+		<div class="col mb-4">
+			<div class="card">
+				<div class="card-body">
+					<h5 class="card-title">{{ favorite_article['title'] }}</h5>
+					<h6 class="card-subtitle mb-2 text-muted">{{ favorite_article['category'] }}</h6>
+					<p class="card-text">{{ favorite_article['description'] }}</p>
+					<a href="{{ favorite_article['route'] }}" class="card-link">Read More</a>
+				</div>
+			</div>
+		</div>
+		{% endfor %}
+	</div>
+</div>
+{% endif %}
+
+{% if help_article_list %}
+<div class='container mb-5'>
+	<h3 class="h3 pb-3">{{ _("Help Articles") }}</h3>
+	<div class="row row-cols-1 row-cols-md-3">
+		{% for item in help_article_list %}
+		<div class="col mb-4 text-truncate">
+			<h5 class="list">{{ item['category'].name }}</h5>
+			<div class="list">
+				{% for article in item['articles'] %}
+				<a href="{{ article.route }}" class="d-block">{{ article.title }}</a>
+				{% endfor %}
+			</div>
+		</div>
+		{% endfor %}
+	</div>
+</div>
+{% endif %}
+
+{% if get_started_sections %}
+	{% for section in get_started_sections %}
+	<div class="container mb-30">
+		<h2>{{ section["name"] }}</h2>
+		{% for item in section["items"] %}
+		<div class="m-20-0">
+			<a href="{{ item.link }}"><b>{{ item.title }}</b></a>
+			{% if item.description -%}
+			<p>{{ item.description }}</p>
+			{%- endif %}
+		</div>
+		{% endfor %}
+		<p><a href="/kb/support">{{ _("See All Articles") }}</a></p>
+		<hr>
+	</div>
+	{% endfor %}
+{% endif %}
+
+{% if topics %}
+<div class="container mb-30">
+	<h2>{{ _("Forum Activity") }}</h2>
+	{% for topic in topics %}
+	<div class="m-20-0">
+		<a href="{{ topic[post_params.link] }}">
+			<b>{{ topic[post_params.title] }}</b>
+		</a>
+		{% if topic[post_params.description] -%}
+		<p>{{ topic[post_params.description] }}</p>
+		{%- endif %}
+	</div>
+	{% endfor %}
+	<p><a href="{{ forum_url }}">{{ _("Visit the forums") }}</a></p>
+	<hr>
+</div>
+{% endif %}
+
+{% if issues | len > 0 -%}
+<div class="container mb-30">
+	<h2>{{ _("Your tickets") }}</h2>
+		{% for doc in issues %}
+			{% include "templates/includes/issue_row.html" %}
+		{% endfor %}
+	<p><a href="/issues">{{ _("See all open tickets") }}</a></p>
+	<a href="/issues?new=1" class="btn btn-primary btn-new btn-sm">
+		{{ _("Open a new ticket") }}
+	</a>
+</div>
+{%- endif %}
```

Comment:
If you're removing issues in listing, remove this. 
Else fix issue_row template